packages/bpk-component-breadcrumb/readme.md

```diff
@@ -0,0 +1,59 @@
+# bpk-component-breadcrumb
+
+> Backpack breadcrumb component.
+
+## Installation
+
+```sh
+npm install bpk-component-breadcrumb --save-dev
+```
+
+## Usage
+```js
+import React, { Component } from 'react';
+import BpkBreadcrumb, { BpkBreadcrumbItem } 'bpk-component-breadcrumb';
+
+
+export default class App extends Component {
+  render() {
+    return (
+      <BpkBreadcrumb label="breadcrumb">
+        <BpkBreadcrumbItem
+          href="/"
+          aria-current="page"
```

Comment:
`aria-current` should be on the active breadcrumb item, same as we have in `stories.js`.